### PR TITLE
Rudder-saas <- Master

### PR DESCRIPTION
**Pull request by ItsSudip**
## Description of the change

Rudder-saas <- Master

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by saikumarrs — APPROVED** [review]

**Comment by saikumarrs** [comment]
@ItsSudip Update the PR description.

File: v0/destinations/google_adwords_remarketing_lists/transform.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const {
 
 const hashEncrypt = object => {
   Object.keys(object).forEach(key => {
-    if (hashAttributes.includes(key)) {
+    if (hashAttributes.includes(key) && object[key]) {
```
Comment (saikumarrs) [inline v0/destinations/google_adwords_remarketing_lists/transform.js:25]: @ItsSudip Please add test cases for this.

Comment (ItsSudip) [reply v0/destinations/google_adwords_remarketing_lists/transform.js:25]: Are you asking for unit test cases?

Comment (saikumarrs) [reply v0/destinations/google_adwords_remarketing_lists/transform.js:25]: Yes

**Review by sanpj2292 — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**